src/main/java/Parser.java

```diff
@@ -0,0 +1,212 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.TextStyle;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Locale;
+
+public class Parser {
+    /**
+     * Gets the relevant task id from command.
+     *
+     * @param command User command.
+     * @return Id of the task to be modified/deleted.
+     */
+    public static Integer[] getTaskIds(String command) {
+        String idString = command.split(" ", 2)[1];
+        String[] ids = idString.split(",");
+        return Arrays.stream(ids)
+                     .map(x -> Integer.parseInt(x) - 1)
```

Comment:
As streams are quite hard to read already, would be good to give meaningful names to variables mapped.